etc/rose.conf.example

```diff
@@ -106,15 +106,23 @@
 #  user-tool=ldap
 [rosa-svn]
 
-# Configuration for Rose Bush server. (Examples are default values.)
+# Configuration for Rose Bush server. (Number examples are default values.)
 [rose-bush]
 #
 ## Cycles list view: default number of cycles per page
 #  cycles-per-page=100
+## An alternate host name. (Default is the server's host name.)
+#  host=NAME
+## E.g.:
+#  host=my-wonderful-host
 ## Job list view: default number of jobs per page
 #  jobs-per-page=15
 ## Job list view: maximum number of jobs per page
 #  jobs-per-page-max=300
+## An alternate service title (Default="Rosie Suites Discovery")
```

Comment:
Given that this is rose-bush "rosie suites discovery" is appropriate as a default - its not about suite discovery.